config/initializers/rack_cors.rb

```diff
@@ -0,0 +1,16 @@
+## Configure Rack CORS Middleware, so that CloudFront can serve our assets.
+## See https://github.com/cyu/rack-cors
+## And see https://stackoverflow.com/a/36585871
+if defined? Rack::Cors
+  Rails.configuration.middleware.insert_before 0, Rack::Cors do
+    allow do
+      origins [
+        # SERVER_DOMAIN should be set to the current env root web address
+        # such as https://staging.idseq.net/
+        ENV["SERVER_DOMAIN"] || "",
```

Comment:
Two suggestions:
1) instead of assigning an empty string here, just leave this entry return `nil` and then compact the array
2) You could use the spread operator here

Ex:
```ruby
[
  ENV["SERVER_DOMAIN"], 
  *Rails.application.config.allowed_cors_origins
].compact
```